dev/deps/spark-deps-hadoop-2.4

```diff
@@ -0,0 +1,185 @@
+JavaEWAH-0.3.2.jar
```

Comment:
yes these are automatically generated, so it's not a huge maintenance cost. I think having them int he repo is good so people can have a definitive reference for what dependencies exist in which package of spark.